src/components/AddLink.js

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+import { getFolderTabs } from '../api';
+import Modal from '../common/Modal/Modal';
+import './AddLink.css';
+import useAsync from './hooks/useAsync';
+
+function AddLink() {
+  const [openModal, setOpenModal] = useState(false);
+  const [modalContent, setModalContent] = useState(null);
+  const [linkInput, setLinkInput] = useState(null);
+  const [tabs, setTabs] = useState();
+  const [, , getFolderTabsAsync] = useAsync(getFolderTabs);
+
+  const fetchData = async () => {
+    const data = await getFolderTabsAsync();
+    setTabs(data);
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);

```

Comment:
`fetchData`라는 함수는 네이밍이 의도,의미를 드러내지 못한다는 점이 있고, 이미 api호출함수를 만들어두셨기 때문에 굳이 매번 렌더링마다 함수를 생성할 필요없이 아래처럼 작성해도되겠습니다. 
부수효과로 setTabs만 해주면 되고, 반환된, resolve된 값을 바로 인자로 전달하면 되니까 표현도 간결하게 가능합니다.

```suggestion
  useEffect(() => {
    getFolderTabsAsync().then(setTabs)
  }, []);
```